test/scripts/agent-images/Containerfile-e2e-base.local

```diff
@@ -41,3 +48,5 @@ RUN useradd -ms /bin/bash user && \
 ## Add your flightctl configuration and certificates
 ADD bin/agent/etc/flightctl/config.yaml /etc/flightctl/
 ADD bin/agent/etc/flightctl/certs/* /etc/flightctl/certs/
+COPY --from=build sosreport/dist/flightctl*.whl /tmp/
+RUN dnf install -y python3-pip && pip install --prefix=/usr /tmp/flightctl*.whl && rm -f /tmp/flightctl*.whl && dnf remove -y python3-pip
```

Comment:
Instead of installing Python, PIP, et al., wouldn't it be better to add the plugin to our [RPM .spec file](https://github.com/flightctl/flightctl/blob/main/packaging/rpm/flightctl.spec) as new `sosreport-plugin` sub-package and then install the new RPM into the test container with the other RPMs (line 22 resp. 29)?